tap_github/client.py

```diff
@@ -231,7 +241,29 @@ def extract_repos_from_config(self):
         """
         repo_paths = list(filter(None, self.config['repository'].split(' ')))
 
-        orgs_with_all_repos = list(filter(lambda x: x.split('/')[1] == '*', repo_paths))
+        visited = set()
+        # Insert the duplicate repos found in the config repo_paths into duplicates
+        duplicates = [x for x in repo_paths if x in visited or (visited.add(x) or False)]
+        if duplicates:
+            LOGGER.warn('Duplicate repositories found: {} and will be synced only once.'.format(duplicates))
+
+        repo_paths = list(set(repo_paths))
+
+        orgs_with_all_repos = []
+        repos_with_errors = []
+        for each_repo in repo_paths:
+            # Split the repo_path by `/`.
```

Comment:
Complete the sentence by providing the reason of the split.

Reply:
Updated the comment